src/test/ui/macros/macro-multiple-matcher-bindings.rs

```diff
@@ -0,0 +1,25 @@
+// Test that duplicate matcher binding names are caught at declaration time, rather than at macro
+// invocation time.
+//
+// FIXME(mark-i-m): Update this when it becomes a hard error.
+
+// compile-pass
+
+#![allow(unused_macros)]
```

Comment:
Hmm, all `unused` warnings should be disabled by default in UI tests, so this shouldn't be necessary.